lane_transform:
  ros__parameters:
    wheelbase: 0.258
    max_speed: 2.2
    min_speed: 0.0
    max_acceleration: 5.0
    min_acceleration: -5.0
    max_delta: 0.314159
    min_delta: -0.314159
    ts: 0.015